src/viewer.js

```diff
@@ -2132,6 +2132,58 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      */
     _cancelPendingImages: function() {
         this._loadQueue = [];
+    },
+
+    /**
+     * Remove the reference strip.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    removeReferenceStrip: function() {
+        this.showReferenceStrip = false;
+
+        if (this.sequenceMode) {
+            if (this.referenceStrip) {
+                this.referenceStrip.destroy();
+                this.referenceStrip = null;
+            }
+        }
+
+        return this;
+    },
+
+    /**
+     * Display the reference strip based on the currently set tileSources.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    addReferenceStrip: function() {
+        this.showReferenceStrip = true;
+
+        if (this.sequenceMode) {
```

Comment:
I've added warning, but only in the test openseadragon.js. Now it's included in the pull request, too.